### PR TITLE
API docs

### PR DESCRIPTION
**Pull request by YunHsiao**
Re: cocos-creator/3d-tasks#2506

Changelog:
 *

### CODE REVIEW

**Comment by pandamicro** [comment]
@YunHsiao Please do not remove Chinese tooltips yet, we will support i18n localization for them, so that you can move both English and Chinese tooltips into the i18n file

**Comment by pandamicro** [comment]
@gameall3d will send a PR to create the i18n file, you can then rebase and fill up in this PR

File: cocos/audio/audio-source-component.ts [hunk 1]
```diff
@@ -56,13 +56,13 @@ export class AudioSourceComponent extends Component {
...
      * @en
-     * The default AudioClip to play
+     * The default AudioClip to play.
      * @zh
      * 设定要播放的音频。
      */
     @property({
         type: AudioClip,
-        tooltip:'设定要播放的音频',
+        tooltip: 'The default AudioClip to play',
```
Comment (VisualSJ) [inline cocos/audio/audio-source-component.ts]: 这个可以放到 editor/i18n 文件夹里
然后这里用 i18n:xxxx.xxxx
编辑器这里稍后支持。直接删了中文的话，后面弄回来又要蛋疼一次了

File: cocos/audio/assets/clip.ts [hunk 2]
```diff
@@ -42,6 +42,10 @@ export const AudioType = Enum({
 });
 
 /**
+ * @en
+ * The audio clip asset. Every clip handles the low-level affairs itself.
```
Comment (CocosLuke) [inline cocos/audio/assets/clip.ts]: * The audio clip asset. Every clip is handled as a low-level event.

File: cocos/audio/audio-source-component.ts (hunk 1 above)
Comment (CocosLuke) [inline cocos/audio/audio-source-component.ts]: Set the default AudioClip to play

File: cocos/audio/audio-source-component.ts [hunk 3]
```diff
@@ -79,7 +79,7 @@ export class AudioSourceComponent extends Component {
      * 是否循环播放音频？
```
Comment (CocosLuke) [inline cocos/audio/audio-source-component.ts:78]: Do you want audio looping of AudioSource?

File: cocos/audio/audio-source-component.ts [hunk 4]
```diff
@@ -56,13 +56,13 @@ export class AudioSourceComponent extends Component {
 
     /**
      * @en
-     * The default AudioClip to play
+     * The default AudioClip to play.
```
Comment (CocosLuke) [inline cocos/audio/audio-source-component.ts]: The default AudioClip plays.

File: cocos/core/3d/framework/batched-skinning-model-component.ts [hunk 5]
```diff
@@ -76,6 +99,10 @@ export class SkinningModelUnit {
         return this._size;
     }
 
+    /**
+     * @en Convevient setter, copying all the necessary information from target skinning model component.
```
Comment (CocosLuke) [inline cocos/core/3d/framework/batched-skinning-model-component.ts]: @en Convenient setter, copying all necessary information from target SkinningModelComponent.

File: cocos/core/3d/framework/batched-skinning-model-component.ts [hunk 6]
```diff
@@ -89,27 +116,41 @@ export class SkinningModelUnit {
...
     public atlasSize: number = 1024;
+
+    /**
+     * @en Size of the generated texture atlas.
+     * @zh 材质中真正参与合图的贴图属性，不参与的属性统一使用第一个 unit 的贴图。
+     */
     @property({
         type: [CCString],
-        tooltip: '材质中真正参与合图的贴图属性，不参与的属性统一使用第一个 unit 的贴图',
+        tooltip: '',
```
Comment (CocosLuke) [inline cocos/core/3d/framework/batched-skinning-model-component.ts]: tooltip: 'The texture attributes that actually participate in the texture of the material, the attributes that are not involved use the texture of the first unit uniformly,'

File: cocos/core/3d/framework/camera-component.ts [hunk 7]
```diff
@@ -139,7 +135,7 @@ export class CameraComponent extends Component {
     }
 
     /**
-     * @en The priority of the camera, it cannot be modified at runtime, instead, it should be set in editor.
+     * @en Render priority of the camera, should be statically set in editor. Dynamically changing this property at runtime won't work.
```
Comment (CocosLuke) [inline cocos/core/3d/framework/camera-component.ts]: * @en Render priority of the camera, should be statically set in editor. You cannot dynamically change this property at runtime.

File: cocos/core/3d/framework/camera-component.ts [hunk 8]
```diff
@@ -292,7 +288,7 @@ export class CameraComponent extends Component {
     }
 
     /**
-     * @en The screen viewport of the camera wrt. sceen size
+     * @en Screen viewport of the camera wrt. sceen size.
```
Comment (CocosLuke) [inline cocos/core/3d/framework/camera-component.ts]: * @en Screen viewport of the camera wrt. The screen size.

File: cocos/core/3d/framework/camera-component.ts [hunk 9]
```diff
@@ -308,8 +304,8 @@ export class CameraComponent extends Component {
     }
 
     /**
-     * @en The scale of the interal buffer size,
-     * set to 1 to keep the same with the canvas size
+     * @en Scale of the interal buffer size,
```
Comment (CocosLuke) [inline cocos/core/3d/framework/camera-component.ts]: * @en Scale of the internal buffer size,

File: cocos/core/3d/framework/renderable-component.ts [hunk 10]
```diff
@@ -141,9 +141,8 @@ export class RenderableComponent extends Component {
     }
 
     /**
-     * @en Returns the material instance corresponding to the sequence number
-     * @zh 获取相对应序号的材质实例。
-     * @param idx Look for the material list number
+     * @en Get material instance of the specified sub-model.
```
Comment (CocosLuke) [inline cocos/core/3d/framework/renderable-component.ts]: * @en Get the material instance of the specified sub-model.

File: cocos/core/3d/framework/renderable-component.ts [hunk 11]
```diff
@@ -161,9 +160,8 @@ export class RenderableComponent extends Component {
     }
 
     /**
-     * 设置对应序号的材质实例
-     * @param index 材质序号
-     * @param matInst 材质实例
+     * @en Set material instance of the specified sub-model.
```
Comment (CocosLuke) [inline cocos/core/3d/framework/renderable-component.ts]: * @en Set the material instance of the specified sub-model.

File: cocos/core/3d/framework/sphere-light-component.ts [hunk 12]
```diff
@@ -79,11 +81,12 @@ export class SphereLightComponent extends LightComponent {
...
 
     /**
-     * 指定光通量或亮度。
+     * @en The photometric term currently using.
+     * @zh 当前使用的光度学计量单位。
      */
     @property({
         type: PhotometricTerm,
-        tooltip: '指定光通量或亮度',
+        tooltip: 'The photometric term currently using',
```
Comment (CocosLuke) [inline cocos/core/3d/framework/sphere-light-component.ts]: tooltip: 'The photometric term currently being used',

File: cocos/core/3d/framework/spot-light-component.ts [hunk 13]
```diff
@@ -98,12 +101,12 @@ export class SpotLightComponent extends LightComponent {
...
     /**
      * @en
-     * The light size, used for spot and point light
+     * The light size, used for spot and point light.
      * @zh
      * 针对聚光灯和点光源设置光源大小。
      */
     @property({
-        tooltip: '针对聚光灯和点光源设置光源大小',
+        tooltip: 'The light source size, used for spot and point light',
```
Comment (CocosLuke) [inline cocos/core/3d/framework/spot-light-component.ts]: tooltip: 'The light source size, used for spot and point lights',

File: cocos/core/3d/framework/spot-light-component.ts [hunk 14]
```diff
@@ -115,12 +118,12 @@ export class SpotLightComponent extends LightComponent {
...
     /**
      * @en
-     * The light range, used for spot and point light
+     * The light range, used for spot and point light.
      * @zh
      * 针对聚光灯和点光源设置光源范围。
      */
     @property({
-        tooltip: '针对聚光灯和点光源设置光源范围',
+        tooltip: 'The light source range, used for spot and point light',
```
Comment (CocosLuke) [inline cocos/core/3d/framework/spot-light-component.ts]: tooltip: 'The light source range, used for spot and point lights',

File: cocos/core/gfx/buffer.ts [hunk 15]
```diff
@@ -67,62 +48,58 @@ export interface IGFXBufferInfo {
 }
 
 /**
- * @zh
- * GFX缓冲。
+ * @zh GFX buffer.
```
Comment (CocosLuke) [inline cocos/core/gfx/buffer.ts]: * @en GFX buffer.

File: cocos/core/gfx/device.ts [hunk 16]
```diff
@@ -367,196 +358,187 @@ export abstract class GFXDevice {
...
 
     /**
-     * @zh
-     * 拷贝缓冲到纹理。
+     * @en Copy buffers to texture.
+     * @zh 拷贝缓冲到纹理。
      * @param buffers 缓冲数组。
-     * @param texture GFX纹理。
-     * @param regions GFX缓冲纹理拷贝区域信息。
+     * @param texture GFX 纹理。
```
Comment (CocosLuke) [inline cocos/core/gfx/device.ts]: English???

File: cocos/core/gfx/device.ts [hunk 17]
```diff
@@ -367,196 +358,187 @@ export abstract class GFXDevice {
...
 
     /**
-     * @zh
-     * 拷贝图像到纹理。
+     * @en Copy texture images to texture.
+     * @zh 拷贝图像到纹理。
      * @param texImages 图像数据源。
-     * @param texture GFX纹理。
-     * @param regions GFX缓冲纹理拷贝区域信息。
+     * @param texture GFX 纹理。
```
Comment (CocosLuke) [inline cocos/core/gfx/device.ts]: English??

File: cocos/core/gfx/device.ts [hunk 18]
```diff
@@ -367,196 +358,187 @@ export abstract class GFXDevice {
...
 
     /**
-     * @zh
-     * 拷贝帧缓冲到缓冲。
+     * @en Copy frame buffer to buffer.
+     * @zh 拷贝帧缓冲到缓冲。
      * @param srcFramebuffer 源帧缓冲。
      * @param dstBuffer 目的缓冲。
-     * @param regions GFX缓冲纹理拷贝区域信息。
+     * @param regions GFX 缓冲纹理拷贝区域信息。
```
Comment (CocosLuke) [inline cocos/core/gfx/device.ts]: English??

File: cocos/core/renderer/models/skinning-model.ts [hunk 19]
```diff
@@ -50,6 +50,12 @@ interface IJointsInfo {
     boundsInfo: aabb[] | null;
 }
 
+/**
+ * @en
+ * Skinning model using baked animation.
```
Comment (CocosLuke) [inline cocos/core/renderer/models/skinning-model.ts]: * The skinning model that is using baked animation.

**Review by CocosLuke — COMMENTED** [review]
Here are some changes, most are new wordings, but some include the question of if there should be English in that comment. Thanks.

File: cocos/core/gfx/binding-layout.ts [hunk 20]
```diff
@@ -40,62 +28,33 @@ export class GFXBindingUnit {
...
 
     /**
-     * @zh
-     * 在指定的binding位置上绑定缓冲。
-     * @param binding 绑定GFX组件的插槽。
-     * @param buffer GFX缓冲。
+     * @en Bind buffer to the specified binding unit.
+     * @zh 在指定的 binding 位置上绑定缓冲。
+     * @param binding 绑定 GFX 组件的插槽。
+     * @param buffer GFX 缓冲。
```
Comment (pandamicro) [inline cocos/core/gfx/binding-layout.ts]: please use English in param, the tool doesn't support localization for @param

**Review by pandamicro — CHANGES_REQUESTED** [review]

File: cocos/audio/audio-source-component.ts (hunk 1 above)
Comment (YunHsiao) [reply cocos/audio/audio-source-component.ts]: These comments are for both getter & setter, so I think only the names should be mentioned.

**Comment by YunHsiao** [comment]
Sorry for the late update. Fixed aforementioned english parameter, tooltip i18n and wordings.

**Review by pandamicro — APPROVED** [review]

**Pull request merged**